backend/controllers/eventController.js

```diff
@@ -0,0 +1,82 @@
+const Event = require("../models/event.model");
+
+module.exports.create_event = async (req, res) => {
+    try {
+        const {event = {}} = req.body
+        const result = await Event.create(event)
+        res.status(201).json({
+            result,
+            message: "OK",
+        });
+    } catch (err) {
+        const errors = _handleErrors(err)
+        res.status(500).json({
+            message: err.message || "Something went wrong",
+            errors
+        });
+    }
+}
+
+module.exports.get_event = async (req, res) => {
+    try {
+        const {query, sort = {createdAt: -1}, limit} = req.body.options || {}
+        const result = await Event.find(query)
+            .sort(sort)
+            .limit(limit)
```

Comment:
There are some breaking changes! Before merging the PR changes needs to be addressed on front-end!
After the required changes has been done on the client, this can be merged!
BTW
Great work! 
LGTM